PIL/Image.py

```diff
@@ -1426,6 +1503,7 @@ def alpha_composite(self, im, dest=(0,0), source=(0,0)):
         self.paste(result, box)
 
     def point(self, lut, mode=None):
+        # type: (Union[List, Callable[[Any], Any]], Optional[Text]) -> Image
```

Comment:
Out of curiosity -- i'm not familiar with what you're describing with `[Any]`. Is it meant to be `List[Any]`?